starter/GitHubSearchTests/GitHubSearchTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+
+import ComposableArchitecture
+
+@testable import GitHubSearch
+
+@MainActor
+final class GitHubSearchTests: XCTestCase {
+  func test_repoSearchResults_whenSearchButtonTapped() async {
+  }
```

Comment:
```suggestion
  func test_user_has_repoSearchResults_when_search() async {
    // Arrange
    // TestStore를 생성합니다.

    // Act & Assert
    // store.send(.keywordChanged("Swift")) 할 때 
    // send.keyword가 "Swift"인지 테스트합니다.
  }
```

테스트케이스를 유저시나리오에 맞춰서 수정하는건 어떨까요?

그리고 테스트에도 간단한 주석을 추가하면 좋을 것 같습니다.